CONTRIBUTING.md

```diff
@@ -34,6 +34,19 @@ Use phrases like "find the flag", "run this command", "download the tool".
 Use [draw.io](https://app.diagrams.net/) to create diagrams.
 If using external images / diagram, make sure they use a CC BY-SA license and give credits (mention author and / or add link to the image source).
 
+## Slides
+
+Slides are to be written in Markdown, using [`reveal-md`](https://github.com/webpro/reveal-md), itself based on [`reveal-js`](https://revealjs.com/).
+Use `reveal-md` and `reveal-js` specifics to split information in slides.
+Aim to make slides attractive, sleek and simple to follow.
+
+Images and diagrams would ideally be animated on slides.
+Aim to use `reveal.js` features to animate drawing of diagrams.
+
+If `reveal.js` drawing is difficult, use [draw.io](https://app.diagrams.net/) to create diagrams.
+Ideally you would "animate" those diagrams by creating multiple incremental versions of the diagram and adding each to a slide;
+when browsing slides pieces of these diagrams will "appear" and complete the final image, rendering an animation-like effect.
+
```

Comment:
This part should also be duplicated on the template repo [1] (together with the commit guides and PR template) and OS [2] repos.

[1] https://github.com/open-education-hub/oer-template/blob/main/CONTRIBUTING.md
[2] https://github.com/cs-pub-ro/operating-systems/blob/main/CONTRIBUTING.md

Reply:
Working on it.

Reply:
@teodutu : https://github.com/open-education-hub/oer-template/pull/13